docstrings_to_openapi/to_openapi.py

```diff
@@ -0,0 +1,55 @@
+import json
+import sys
+
+import yaml
+
+from .cli import OPTS
+
+
+def main():
+    """The main function to bulding the OpenAPI doc for the app's API.
+
+    Here starts by mapping the supporting frameworks and docstring styles that
```

Comment:
It's a bit difficult to understanding this sentence, do you mean something like this?

“It starts by identifying supported frameworks and docstring styles which are then used to parse controller/view docs.”

Reply:
Sure i can edit it, the top line i usually put there as a summary for the docstring